docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md

```diff
@@ -95,37 +122,35 @@ $ sudo rpm -e nebula-graph-explorer-<version>.x86_64
 2. 使用 `tar -xvf` 解压 tar 包。
 
    ```bash
-   tar -xvf nebula-graph-explorer-<version>.tar.gz
+   $ tar -xvf nebula-graph-explorer-<version>.tar.gz
    ```
 
-### 部署
-
-!!! Note
-
-    根目录 nebula-graph-explorer 下一共有两个安装包：nebula-graph-explorer 和 nebula-http-gateway。用户需要在同一台机器上分别部署并启动服务，才能完成 Explorer 的部署。
-
-1. 部署 nebula-http-gateway 并启动。
+3. 拷贝 License 至`nebula-explorer`目录下。
 
    ```bash
-   $ cd nebula-http-gateway
-   $ nohup ./nebula-httpd &
+   $ cp -r <license> <explorer_path>
    ```
 
-2. 部署 nebula-graph-explorer
+  !!! enterpriseonly
 
-   ```bash
-   $ cd nebula-graph-explorer
-   $ npm run start
-   ```
+        License 仅在企业版提供，请发送邮件至inquiry@vesoft.com。
+
+4. 启动explorer。
+
+  ```bash
+  $ ./nebula-httpd &
+  ```
+  当屏幕返回以下信息时，表示 Explroer 已经成功启动。
+  ```bash
+  [1] 5169
```

Comment:
`[1] 5169` is not a fixed output, is it better to add `for example` before the output?